.github/workflows/generate_missing_mappings.yml

```diff
@@ -0,0 +1,113 @@
+name: Generate and commit taxonomy mapping changes to requested PR
+
+on:
+  issue_comment:
+    types: [created]
+
+concurrency:
+  group: "generate_mappings"
+  cancel-in-progress: false
+
+jobs:
+  generate_missing_mappings:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: read
+    if: ${{github.event.issue.pull_request && github.event.comment.body == '/generate_mappings'}}
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Checkout PR
+      run: gh pr checkout $ISSUE
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        ISSUE: ${{ github.event.issue.html_url }}
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+    - name: Set up Podman
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install podman
+    - name: Launch Qdrant server
+      run: |
+        port=6333
+        podman run -d -p ${port}:${port} qdrant/qdrant
+        echo "Waiting for Qdrant server to be ready..."
+        sleep 25
+    - uses: cue-lang/setup-cue@v1.0.0
+      with:
+        version: 'v0.7.0'
+    - name: Generate distribution files to gather all published mappings in json format
+      run: make VERBOSE=1
+    - name: Generate missing mappings
+      env:
+        OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        OPENAI_API_BASE: https://openai-proxy.shopify.ai/v1
+      run: |
+        make generate_mappings
+    - name: Clean cache and re-generate mapping distribution files
+      run: |
+        make clean
+        make VERBOSE=1
```

Comment:
Curious why we need a make build before and after the generate mappings call?

Reply:
Maybe better to use `bin/generate_dist` directly with a targets flag for the second call? And the initial one could be `make seed` to save running generation logic when we don't need it?

Reply:
> use bin/generate_dist directly with a targets flag for the second call?

The `make generate_mappings` command in the initial stage modifies yml files in the `data/integrations` directory. However, the follow-up `bin/generate_dist` command doesn't seed these recent changes because it's using a cached version of the database.

> the initial one could be `make seed`

Sounds good.

Reply:
Done.